src/macros.jl

```diff
@@ -233,36 +233,88 @@ function assert_validmodel(m, macrocode)
     end
 end
 
-function _canonicalize_sense(sns::Symbol)
-    if sns == :(==)
-        return (:(==),false)
-    elseif sns == :(>=) || sns == :(≥)
-        return (:(>=),false)
-    elseif sns == :(<=) || sns == :(≤)
-        return (:(<=),false)
-    elseif sns == :(.==)
-        return (:(==),true)
-    elseif sns == :(.>=) || sns == :(.≥)
-        return (:(>=),true)
-    elseif sns == :(.<=) || sns == :(.≤)
-        return (:(<=),true)
+function _check_vectorized(sense::Symbol)
+    sense_str = string(sense)
+    if sense_str[1] == '.'
+        Symbol(sense_str[2:end]), true
     else
-        error("Unrecognized sense $sns")
+        sense, false
     end
 end
 
 # two-argument constructconstraint! is used for one-sided constraints.
 # Right-hand side is zero.
-function sense_to_set(sense::Symbol)
-    if sense == :(<=)
-        return MOI.LessThan(0.0)
-    elseif sense == :(>=)
-        return MOI.GreaterThan(0.0)
+sense_to_set(_error::Function, ::Union{Val{:(<=)}, Val{:(≤)}}) = MOI.LessThan(0.0)
+sense_to_set(_error::Function, ::Union{Val{:(>=)}, Val{:(≥)}}) = MOI.GreaterThan(0.0)
+sense_to_set(_error::Function, ::Val{:(==)}) = MOI.EqualTo(0.0)
+sense_to_set(_error::Function, ::Val{S}) where S = _error("Unrecognized sense $S")
+
+function parseoneoperatorconstraint(_error::Function, vectorized::Bool, ::Val{:in}, aff, set)
```

Comment:
`parse_one_operator_constraint`. This isn't the right PR to discuss JuMP's policy on underscores, but I can't approve `parseoneoperatorconstraint`.